units/units.cpp

```diff
@@ -231,7 +231,7 @@ static const umap base_unit_names{
     {unit_cast(precise::special::ASD), "ASD"},
     {unit_cast(precise::special::rootHertz), "rootHertz"},
     {currency, "$"},
-    {count, "item"},
+    {count, "count"},
```

Comment:
There is an inconsistency now:
- `count` formats as `"counts"`
- `count/m` formats as `"item/m"`

Is this deliberate?